conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -413,16 +448,34 @@ class OrganizationMetadataEntitySetsService(
             organizationId = organizationId,
             entityTypeId = columnsEntityTypeId,
             name = buildColumnEntitySetName(organizationId),
-            _title = "Datasets for $organizationId",
-            _description = "Datasets for $organizationId",
+            _title = "Dataset columns for $organizationId",
+            _description = "Dataset columns for $organizationId",
             contacts = mutableSetOf(),
             flags = EnumSet.of(EntitySetFlag.METADATA)
     )
 
-    private fun buildOrganizationMetadataEntitySetName(organizationId: UUID): String = quote(
-            "org-metadata-$organizationId"
-    )
-
+    private fun buildOrganizationMetadataEntitySetName(organizationId: UUID): String = quote("org-metadata-$organizationId")
```

Comment:
why is this wrapped in quotes?

Reply:
(same question for the other entity set names)

Reply:
I don't know. Git history shows the quotes have been there for since the monorepo, so I assume they've been there since this class was introduced.

Reply:
hm I think that's a bug actually -- this will cause the entity set names to be stored with quotes as part of the name, which is definitely not intended and is likely to cause issues with a bunch of external tools. If the name is only used for entity set initialization, you should probably remove that.

Reply:
It does look weird to me, happy to remove if you think it's a bug.

Reply:
yeah that would be ideal especially if you're recreating them all -- it is a bug.